packages/v4/patternfly-docs/content/design-guidelines/components/expandable-section/expandable-section.md

```diff
@@ -23,7 +23,7 @@ Use an expandable section with a blue border to add more visual weight. This var
 <img src="./img/disclosure.png" alt="Disclosure example" width="1060"/>
 
 
-## Examples
+
```

Comment:
Would add a H3 under "Variations" that says "Expandable section with blue line" or something cause it's kinda floating there on its own right now